test/spmd/test_xla_sharding.py

```diff
@@ -126,17 +125,14 @@ def test_optimizer_step_with_sharding(self):
       loss.backward()
       optimizer.step()
       xm.mark_step()
+      # After functionalization, in-place ops like tensor.add_ are no longer availble. Therfore, we need
+      # a new way to carry the sharding spec between iterations. This is a tentative fix.
+      xs.mark_sharding(model.fc1.weight, self._get_mesh((1, self.n_devices)),
```

Comment:
the purpose of this test is to make sure in a regular pytorch training case, user only need to call  `mark_sharding` once on weight once. This is currently broken. Since we don't want to ask user to add `mark_sharding` on every step, we can just leave this test disabled.

Reply:
Got you. I'm fine of skipping the tests as it is. This is an attempt trying to provide a way to workaround the test.